### ISSUE TITLE
5.0.2 on Mac does not hold viewpoint when scrolling if new posts occur

### ISSUE THREAD

**Issue by dsinn23** [h3poteto/whalebird-desktop#4233]
## Description
Since installing 5.0.x on Mac, I have noticed that when looking at a timeline, Whalebird no longer holds the scroll point as new posts occur. This is most evident in fast updated timelines like the overall federated one. This did not occur in the 4.x release train.

## How To Reproduce
1. Selected Federated timeline
2. Notice post that is interesting
3. Scroll to read the post
4. Release the mouse
5. Notice the post scrolls off the screen due to new posts to the timeline

## Your Environment
 - OS: MacOS x86 13.2.1
 - Whalebird Version: 5.0.2 (and 5.0.1)
 - Instance: techhub.social

**Comment by JonathanDoughty** [comment]
Yup; just installed WhaleBird successfully for the first time moments ago. (The first time I tried via brew a couple weeks ago I was getting SHA mismatches so back burner-ed a trial. I hit this within moments. WhaleBird on first experience seems promising but this was obvious right after I switched from my main timeline and made it pretty much unusable.

MacOS M1 13.3.1
WhaleBird 5.0.3
Instance: mastodon.social
